homeassistant/helpers/selector.py

```diff
@@ -79,27 +79,27 @@ def serialize(self) -> dict[str, dict[str, _T]]:
         return {"selector": {self.selector_type: self.config}}
 
 
-SINGLE_ENTITY_SELECTOR_CONFIG_SCHEMA = vol.Schema(
+ENTITY_FILTER_SELECTOR_CONFIG_SCHEMA = vol.Schema(
     {
         # Integration that provided the entity
         vol.Optional("integration"): str,
         # Domain the entity belongs to
-        vol.Optional("domain"): vol.Any(str, [str]),
```

Comment:
You're removing `domain` here, but it's still part of `EntityFilterSelectorConfig` class

Reply:
let's add a test to verify domain still works 👍